bolt-http4k/src/main/java/test_locally/Http4kSlackApp.java

```diff
@@ -0,0 +1,65 @@
+package test_locally;
+
+import com.slack.api.bolt.App;
+import com.slack.api.bolt.request.RequestHeaders;
+import com.slack.api.bolt.util.SlackRequestParser;
+import kotlin.Pair;
+import kotlin.jvm.functions.Function1;
+import org.http4k.core.Request;
+import org.http4k.core.Response;
+import org.http4k.core.Status;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.util.Collections.singletonList;
+import static java.util.stream.Collectors.toMap;
+import static org.http4k.core.ParametersKt.toParameters;
+import static org.http4k.core.Status.INTERNAL_SERVER_ERROR;
+
+public class Http4kSlackApp implements Function1<Request, Response> {
+
+    private final App app;
+    private final SlackRequestParser requestParser;
+
+    public Http4kSlackApp(App app) {
+        this.app = app;
+        requestParser = new SlackRequestParser(app.config());
+    }
+
+    @Override
+    public Response invoke(Request request) {
+        try {
+            return toHttp4kResponse(app.run(toSlackRequest(request)));
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Ah, I overlooked this. Can you add 

```java
    private static final Logger LOGGER = LoggerFactory.getLogger(Http4kSlackApp.class);
```

and then, update this line?

```suggestion
            LOGGER.error(e.getMessage(), e);
```